src/charts/trends/trends-chart.directive.js

```diff
@@ -82,10 +82,17 @@
          <button ng-click="addDataPoint()">Add Data Point</button>
        </div>
      </div>
-   </div>
+     <hr class="col-md-12">
+     <div class="col-md-12">
+       <div pf-card head-title="Cluster Utilization" show-top-border="true"
+            footer="footerConfig" filter="filterConfig">
+         <div pf-trends-chart config="config2" chart-data="data"></div>
+       </div>
+     </div>
+    </div>
  </file>
  <file name="script.js">
- angular.module( 'patternfly.charts' ).controller( 'ChartCtrl', function( $scope ) {
+ angular.module( 'test', ['patternfly.charts', 'patternfly.card'] ).controller( 'ChartCtrl', function( $scope ) {
```

Comment:
For the ngDoc examples, whenever two directives are needed, they need to be listed in the module dependencies.  In this case, I wrapped a trends chart inside a card.